### PR TITLE
Update annual-report-2022.md

### PR DESCRIPTION
**Pull request by shaneutt**
Trying to fill in at least some of the 2022 report.

### CODE REVIEW

File: sig-network/annual-report-2022.md [hunk 1]
```diff
@@ -4,17 +4,14 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   -Gateway API reached BETA status
```
Comment (MikeZappa87) [inline sig-network/annual-report-2022.md:7]: We should be consistent with spaces between the dash and text

**Comment by rikatz** [comment]
/cc @rikatz @strongjz 
Let's try to put some news on ingress-nginx, as a subproject here

**Comment by shaneutt** [comment]
> /cc @rikatz @strongjz Let's try to put some news on ingress-nginx, as a subproject here

Sure, throw me some highlights

**Comment by shaneutt** [comment]
Calling this ready to review as an incremental update.

**Comment by shaneutt** [comment]
This one popped back up on my radar: starting to get close to 2024 now, should we merge what we have here at least? Should we do something different?

**Pull request closed without merging**